test/testcases/block/06_codeblock/with_lang_in_fenced_block_name_with_dash.options

```diff
@@ -0,0 +1,2 @@
+:enable_coderay: false
+
```

Comment:
I think there shouldn't be any need for this file anymore since you modify the language name for coderay to use underscores instead of dashes.